warehouse/upload.go

```diff
@@ -1716,7 +1731,6 @@ func (job *UploadJobT) setUploadError(statusError error, state string) (string,
 
 	// On aborted state, validate credentials to allow
 	// us to differentiate between user caused abort vs platform issue.
-	state = model.Aborted
```

Comment:
This got pushed by mistake https://github.com/rudderlabs/rudder-server/pull/2956